modules/installing-rhv-downloading-rhcos-image.adoc

```diff
@@ -20,22 +20,14 @@ IMPORTANT: Do not substitute the {op-system} image specified by these instructio
 $ mkdir rhcos
 $ cd rhcos
 ----
-
-. In a browser, go to link:https://github.com/openshift/installer/blob/release-4.4/data/data/rhcos.json[].
```

Comment:
@rolfedh while at it, can you suggest a way to replace the hard-coded `4.4` with a more future-resilient `{product-version}` ?